src/extension.ts

```diff
@@ -3,38 +3,57 @@ import * as vscode from 'vscode';
 import WebSocket from 'ws';
 import QuickPickItem = vscode.QuickPickItem;
 import QuickPickOptions = vscode.QuickPickOptions;
-import { fixRemoteUrl, getURL } from './utils';
+import * as utils from './utils';
 
 export function activate(context: vscode.ExtensionContext) {
+    context.subscriptions.push(vscode.commands.registerCommand('devtools-for-chrome.attach', async () => {
+        attach(context);
+    }));
 
-    context.subscriptions.push(vscode.commands.registerCommand('devtools-for-chrome.start', async () => {
-        const settings = vscode.workspace.getConfiguration('vscode-devtools-for-chrome');
-        const hostname = settings.get('hostname') as string || 'localhost';
-        const port = settings.get('port') as number || 9222;
+    context.subscriptions.push(vscode.commands.registerCommand('devtools-for-chrome.launch', async () => {
+        launch(context);
+    }));
+}
 
-        const checkDiscoveryEndpoint = (url: string) => {
-            return getURL(url, { headers: { Host: 'localhost' } });
-        };
+const address = 'localhost';
```

Comment:
Shouldn't these be read from the settings instead of hard coded?

Reply:
I had them hardcoded as launching only works locally (as in Chrome is installed locally). Remote launching would be a different kinda feature.